core/logger/sentry.go

```diff
@@ -22,6 +22,17 @@ const (
 )
 
 func init() {
+	initSentry()
+}
+
+func initSentry() {
```

Comment:
Leftover from testing?
```suggestion
```

Reply:
Nope, I just wanted to clean up the `init` function and keep all the sentry stuff clear

Reply:
There are already multiple init funcs, and this one in `sentry.go` is sentry only.